.github/workflows/tests.yml

```diff
@@ -25,21 +25,26 @@ jobs:
         include:
           - RUN: 1
             DRUPAL_TESTING_DRUPAL_VERSION: '~9.3'
+            DRUPAL_TESTING_COMPOSER_PROJECT_VERSION: '9.3.0'
```

Comment:
It concerns me that we have both of these - or more to the point - that they can get out of step. But OTOH it allows the thunder-project to test interesting stuff. But something feels icky.

Reply:
Addressed by defaulting `DRUPAL_TESTING_COMPOSER_PROJECT_VERSION` to `DRUPAL_TESTING_DRUPAL_VERSION` after discussion with @dbosen